shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/src/main/antlr4/imports/opengauss/DDLStatement.g4

```diff
@@ -1161,7 +1161,6 @@ alterTypeClauses
     | RENAME TO name
     | RENAME ATTRIBUTE name TO name dropBehavior?
     | SET SCHEMA name
-    | SET LP_ operatorDefList RP_
```

Comment:
Does openGauss not support this syntax?

Reply:
Yes, It isnt mentioned in their syntax for `ALTER TYPE`

You can see it here : https://opengauss.org/en/docs/3.0.0/docs/Developerguide/alter-type.html

Reply:
> Yes, It isnt mentioned in their syntax for `ALTER TYPE`
> 
> You can see it here : https://opengauss.org/en/docs/3.0.0/docs/Developerguide/alter-type.html

Ok, I got it.